routes/cube/deck.js

```diff
@@ -284,6 +284,71 @@ router.get('/download/cockatrice/:id/:seat', async (req, res) => {
   }
 });
 
+router.get('/download/topdecked/:id/:seat', async (req, res) => {
+  try {
+    const deck = await Draft.getById(req.params.id);
+    if (!deck) {
+      req.flash('danger', `Deck ID ${req.params.id} not found/`);
+      return res.redirect('/404');
+    }
+    const seat = deck.seats[0];
```

Comment:
you have the `seat` parameter, but only download seat 0. For multiplayer drafts, this means that only player 1 would be able to download their decks